microsoft/onefuzz
# parse stderr for ASAN for libfuzzer targets

## Summary of the Pull Request

Adds parsing stderr to look for ASAN style logs during libfuzzer crash reporting

## PR Checklist
* [X] Applies to work item: #118 

## Info on Pull Request

This PR does two things:

1. Adds parsing of stderr to libfuzzer targets
2. Standardizes run_cmd onto our own Output

This *doesn't* implement stderr parsing for non-libfuzzer tasks, nor does it attempt to parse the output if you used a debugger.  Eventually, it would be good to normalize our debugger wrappers to include the same Output but that is out of scope for this PR.

## Validation Steps Performed

1. Run standard libfuzzer binary and see crash reports
2. Run a test with a go-fuzz based binary and see crash reports
